docs/tools/flow-cli/flix.md

```diff
@@ -96,8 +96,85 @@ pub fun main(): String {
 }
 ```
 
+
+### Cadence Doc Pragma
+It's recommended to use pragma to set the metadata for the script or transaction. More information on [Cadence Doc Pragma FLIP](https://github.com/onflow/flips/blob/main/application/20230406-interaction-template-cadence-doc.md) 
+
+```cadence
+import "HelloWorld"
+
+#interaction (
+    version: "1.1.0",
+    title: "Update Greeting",
+    description: "Update the greeting on the HelloWorld contract",
+    language: "en-US",
+)
+transaction(greeting: String) {
+
+  prepare(acct: AuthAccount) {
+    log(acct.address)
+  }
+
+  execute {
+    HelloWorld.updateGreeting(newGreeting: greeting)
+  }
+}
+
+```
+
+:::info
+Cadence v0.42.7 supports more Cadence docs provides more funcationality. Parameters descriptions will populate human readable title and description. Look forward to `parameters` property which is an array of `Parameter`.
```

Comment:
"supports more Cadence docs provides more funcationality" needs to be rewrote. Also, typo there. Following sentences need some more clarity as well.